README.md

```diff
@@ -214,6 +215,15 @@ spring:
       static-locations: { source folder of static content, e.g. file:/opt/files }
   ```
 
+##### Example Serving DCAT Metadata
+
+Supported file formats: .ttl, .rdf, .nq and .jsonld
+
+  ```yaml
+ldes:
+  dcat: { path of file containing DCAT metadata, e.g. "dcat/metadata.ttl"  }
+  ```
+
```

Comment:
should we mention something in the readme on how to use the templates?

Reply:
We explain it in the readme of the templates but I will refer the templates directory here so that it can be found more easily